packages/contracts/test/LibExperience.t.sol

```diff
@@ -27,4 +28,64 @@ contract LibExperienceTest is MudV2Test {
       assertEq(exp[i], 1);
     }
   }
-}
\ No newline at end of file
+
+  function testGetExp() public {
+    LibExperience.initExp(targetEntity);
+    assertTrue(LibExperience.hasExp(targetEntity));
+
+    uint32[PStat_length] memory addExp = [uint32(1), 1, 1];
+    LibExperience.increaseExp(targetEntity, addExp);
+
+    uint32[PStat_length] memory exp = LibExperience.getExp(targetEntity);
+    for (uint256 i; i < exp.length; i++) {
+      assertEq(exp[i], 1);
+    }
+  }
+
+  function testHasExp() public {
+    assertFalse(LibExperience.hasExp(targetEntity));
+
+    LibExperience.initExp(targetEntity);
+    assertTrue(LibExperience.hasExp(targetEntity));
+  }
```

Comment:
you can remove
```solidity
LibExperience.initExp(targetEntity);
assertTrue(LibExperience.hasExp(targetEntity));
```
from all other tests, since you add a dedicated existence check